agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1809,15 +1809,22 @@ bool backhaul_manager::handle_1905_discovery_query(ieee1905_1::CmduMessageRx &cm
         LOG(ERROR) << "addClass wfa_map::tlvApOperationalBSS failed, mid=" << std::hex << (int)mid;
         return false;
     }
-    //add dummy data
-    auto radio_list     = tlvApOperationalBSS->create_radio_list();
-    auto radio_bss_list = radio_list->create_radio_bss_list();
-    radio_bss_list->set_ssid("wlan_0");
-    radio_list->add_radio_bss_list(radio_bss_list);
-    radio_bss_list = radio_list->create_radio_bss_list();
-    radio_bss_list->set_ssid("wlan_2");
-    radio_list->add_radio_bss_list(radio_bss_list);
-    tlvApOperationalBSS->add_radio_list(radio_list);
+
+    for (auto vaps_list : m_vaps_map) {
+        auto radio_list         = tlvApOperationalBSS->create_radio_list();
+        radio_list->radio_uid() = network_utils::mac_from_string(vaps_list.first);
+        for (auto vap : vaps_list.second.vaps) {
+            if (vap.mac == network_utils::ZERO_MAC)
+                continue;
```

Comment:
For single-lines if, maybe we should agree on either always omitting the braces, or always use them. At the moment it looks like it's up to the developer's preference, which means we may end up with both styles mixed.
If so we should add it to `CODINGSTYLE.md`. clang-format may even be able to detect and/or fix it.

(not for this PR)

Reply:
https://doodle.com/poll/wwyfaymx84chhe3k

Reply:
Unfortunately, [clang-format doesn't support checking this... clang-tidy does support it](https://stackoverflow.com/questions/26111162/can-clang-format-add-braces-to-single-line-if-statements-etc), but that has to be called with all the relevant -I and -D options, so separately in each build type.